samples/highcharts/blog/aapl-price/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: AAPL Price
```

Comment:
It's the same as http://localhost:3030/samples/#view/stock/demo/basic-line, use that instead